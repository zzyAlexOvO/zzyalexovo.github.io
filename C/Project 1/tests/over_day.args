Sichuan Happy 23:01:24