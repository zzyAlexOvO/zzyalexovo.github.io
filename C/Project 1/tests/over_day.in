Sichuan::Happy::02:38:23
Sichuan::Happy::08:01:23
Sichuan::Happy::14:30:23
Sichuan::Happy::20:59:23
Sichuan::Happy::23:01:23