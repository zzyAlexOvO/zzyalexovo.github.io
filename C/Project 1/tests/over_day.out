The next train to Happy from Sichuan departs at 02:38:23
